lib/faker/address/es.ex

```diff
@@ -0,0 +1,734 @@
+defmodule Faker.Address.Es do
+  import Faker, only: [sampler: 2]
+
+  alias Faker.Name
+
+  @geobase32 '0123456789bcdefghjkmnpqrstuvwxyz'
+
+  @moduledoc """
+  Functions for generating addresses in Spanish
+  """
+
+  @doc """
+  Return random building number.
+  """
+  @spec building_number() :: String.t()
+  def building_number do
```

Comment:
if the function is not different from main module `Faker.Address` I don't think we need to override it. The idea is to always use main modules without locale names in the name, while config will determinate which one to call.

Reply:
I see your point 👍